tests/integration/targets/test_mysql_db/tasks/config_overrides_defaults.yml

```diff
@@ -3,13 +3,29 @@
     config_file: "/root/.my1.cnf"
     fake_port: 9999
     fake_host: "blahblah.local"
+    include_dir: "/root/mycnf.d"
 
 - name: Create custom config file
   shell: 'echo "[client]" > {{ config_file }}'
 
 - name: Add fake port to config file
   shell: 'echo "port = {{ fake_port }}" >> {{ config_file }}'
 
+- name: Add blank line
+  shell: 'echo "" >> {{ config_file }}'
+
+- name: Create include_dir
+  file:
+    path: '{{ include_dir }}'
+    state: directory
+    mode: '0777'
+
+- name: Add include_dir
+  lineinfile:
+    path: '{{ config_file }}'
+    line: '!includedir {{ include_dir }}'
+    insertafter: EOF
```

Comment:
```suggestion
    insertafter: EOF
when:  (pymysql_version.stdout | default('1000', true)) is version('0.9.3', '>=')
```

Reply:
This will add the `includedir` line only when the connector is pymysql 0.9.3 or newer, or MySQLdb

Reply:
very nice, thanks!